packages/frontend/src/components/HorizontalSeparator.tsx

```diff
@@ -0,0 +1,16 @@
+import cx from 'classnames'
+import React from 'react'
+export interface SeparatorProps {
+  className?: string
+}
+export function HorizontalSeparator(props: SeparatorProps) {
+  const { className } = props
```

Comment:
We never use this convention anywhere. We do:
- either destructure in the function signature, e.g. `function Component({ a, b }: ComponentProps`
- or just use `props.a` and `props.b` directly

Reply:
You can also update KnowledgeNuggetItem to follow this :) as I missed it during your previous PR

Reply:
Okay, I have to change my habits from previous projects 👍